src/components/chat/item-header.tsx

```diff
@@ -33,6 +50,12 @@ export default function ChatItemHeader({
       <i className="ui icon reply" onClick={onClickReply} />
       <i className="ui icon edit" onClick={onClickEdit} />
       <i className="ui icon remove" onClick={() => onRemove(chat)} />
+      {collapsed && (
+        <span>
+          {' '}
+          {chat.children.length} message{chat.children.length > 1 && 's'} hidden
```

Comment:
`children.length` doesn't count the children's `children` prop. recursive function would be needed to display that data